### DISCUSSION TITLE
How to search for a Svelte 5 `$inspect(…)` rune?

### DISCUSSION THREAD

**Discussion by notramo** [biomejs/biome#4656] [Q&A]
I want to use `biome search` to find all usage of the [`$inspect()`](https://svelte.dev/docs/svelte/$inspect) rune. However, the `$` in the rune name makes the parser treat it as a GritQL variable, and finds every function call as results. I couldn't get far with escaping it, because it finds zero matches when adding an escape char.
I ran the following command (POSIX shells might require different string literal escaping, but it's valid `elvish` syntax)
```
bun run biome search '`\$inspect($_)`'
```

**Comment by ematipico** [comment]
cc @arendjr maybe you know the answer

**Comment by ah-yu** [comment]
Try the following command

```bash
bun run biome search '`$func($_)` where $func <: "$inspect"'
```

**Reply by notramo** [reply]
It throws an error without searching any files. I guess `where` is not implemented yet.

**Reply by dyc3** [reply]
Yeah that query should work. https://app.grit.io/studio?key=EbSM7pWSppixl2wSnc2ig

What error are you getting? IIRC `where` is implemented in biome.

**Reply by notramo** [reply]
Meaningless errors, so I guess it's because of the unsupported `where`.

```
where internalError/io  INTERNAL  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━


$func internalError/io  INTERNAL  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━


<: internalError/io  INTERNAL  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━


"$inspect" internalError/io  INTERNAL  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━


Searched 0 files in 112µs. Found 0 matches.
internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ No files were processed in the specified paths.
```
